cli/src/swam/cli/Server.scala

```diff
@@ -0,0 +1,171 @@
+package swam
+package cli
+
+import scala.collection.mutable.ListBuffer
+
+// Simple server
+import java.io._
+import java.net.{ServerSocket, Socket}
+import java.nio.ByteBuffer
+import java.nio.file.Path
+
+import cats.effect.IO
+import swam.optin.coverage.{CoverageListener, CoverageReporter}
+import swam.runtime.{Function, Value}
+
+import scala.util.control.Breaks._
+
+// TODO: Read port from env-variable SWAM_SOCKET_PORT
```

Comment:
Can we use the config library instead? It's already there, we use [Lightbend config](https://github.com/lightbend/config)

Reply:
Cool, I'll look into it..! Maybe putting SWAM_SOCKET_PORT into the .env is not necessary, since it shouldn't be greatly dependent on the environment.